lib/model/rofolder.go

```diff
@@ -16,113 +16,135 @@ import (
 
 type roFolder struct {
 	stateTracker
+	scan
```

Comment:
I dont get how would like to have it. The same line is in the rwfolder? Do you mean it should different?